pkg/agent/flowexporter/connections/connections.go

```diff
@@ -157,6 +157,8 @@ func (cs *connectionStore) ReleaseConnStoreLock() {
 func (cs *connectionStore) UpdateConnAndQueue(pqItem *flowexporter.ItemToExpire, currTime time.Time) {
 	conn := pqItem.Conn
 	conn.LastExportTime = currTime
+	conn.AppProtocolName = ""
+	conn.HttpVals = ""
```

Comment:
Is this needed?

Reply:
This is required as the conn once sent to the aggregator should no longer contain the l7 events, we may find the fields present when no L7 event is received but the TCP connection is up, thus to take care of such case, we should set the fields back to empty after every export